wordpress-mobile/WordPress-iOS
# [All Time Widget] Add data handling

Ref #13030 

This implements data handling for the 'All Time' widget.
- When Insights are accessed in the WP app, 'All Time' data is stored in `AllTimeData.plist`.
- The widget loads and saves data from/to the plist.
- The widget loads site information from `UserDefaults`, and uses it to fetch stat data.

The data is displayed in plain labels in the widget to facilitate testing this. The real UI will be implemented in a future PR.

To test:

---
- Go to Stats > Insights.
- Select `Widgets` in the nav bar, and then `Use this site`.
- Add the `WordPress All-Time` widget to your Today view.
- Verify the data displayed matches Insights > All-Time in the app.

<img width="400" alt="insights" src="https://user-images.githubusercontent.com/1816888/70186188-934f8380-16a8-11ea-966e-787e0b634fa1.png">

<img width="400" alt="widget" src="https://user-images.githubusercontent.com/1816888/70186229-b11ce880-16a8-11ea-8268-9cc2fcb4bdd0.png">

---
- In the app, switch sites and `Use this site` again to change the site used for widgets.
- Verify the All Time widget updates accordingly.

---
PR submission checklist:
